newIDE/app/src/Export/ExportDialog/index.js

```diff
@@ -237,6 +238,12 @@ const ExportDialog = ({
           authenticatedUser={authenticatedUser}
           game={game}
           onGameUpdated={setGame}
+          getThumbnailURL={(buildId: string) => {
+            if (!project) {
```

Comment:
That looks like a repetition of the function I just osted a comment on, if we follow the DRY code principle, `getWebBuildThumbnailUrl` should probably already check and return undefined if the project argument is undefined.

Reply:
This is a way of seeing this. But, I personally prefer to have a method that ask what it needs and let the caller do what they need to give the right parameters. It's not to `getWebBuildThumbnailUrl` to decide what should happen if there is no project.